### PR TITLE
Branch review ghana

### PR DESCRIPTION
**Pull request by mariarrt94**
Hi @mariaruth 

I created this pull request for the review of the sessions. Here are the PDFs for the first and second sessions:  
- [Introduction to R](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/1-introduction-to-r.pdf)   
- [Data Wrangling](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/2-data-wrangling.pdf)

Let me know if you have any comments. I designed the sessions in a modularized way, so if it feels like too much content at the moment, I can easily adjust or cut parts.  

At the end, I’ll need to review the overall flow, but that shouldn’t involve any major changes.  

Let me know if you have any feedback!  I will share the third and fourth session in the upcoming days.

### CODE REVIEW

**Comment by mariarrt94** [comment]
Hi @mariaruth   

The slides are ready for your review. You will note that the diagrams—which include a picture of Excel, the code, and the output—are still from Luise's sessions. I will work on updating these tomorrow or Wednesday. The diagrams will be changed to match the data we are using, but nothing else changes. 

Let me know if you have any comments. I designed the sessions in a modularized way, so if it feels like too much content at the moment, I can easily adjust or cut parts.

To navigate more easily, here are links to each session (session 1 for day 1 and 2 are ready, but not on github linking them here as well):  

- **Day 1**  
  - [Session 1: Introduction to Government Analytics](https://worldbankgroup-my.sharepoint.com/:p:/g/personal/mjones5_worldbank_org/EWLbooF9jytEkQj7Lv9xIe8BZjD9xlwTbON6oPc8DORT5A?e=15JkRb)  
  - [Session 2: Introduction to Statistical Programming and R](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/1-introduction-to-r.pdf)  

- **Day 2**  
  - [Session 1: Introduction to Government Data Sources & Analytics Methods](https://worldbankgroup-my.sharepoint.com/:p:/g/personal/mjones5_worldbank_org/EQe0bHOoD2tDoo9-ZKaArvoB9L-XSxvHP6jwLa3SvvTyKg?e=zU74Ln)  
  - [Session 2: Data Wrangling in R](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/2-data-wrangling.pdf)  

- **Day 3**  
  - [Session 2: Descriptive Statistics in R](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/3-descriptive-statistics.pdf)  

- **Day 4**  
  - [Session 2: Data Visualization in R](https://github.com/worldbank/dime-r-training/blob/branch_review_ghana/Presentations-Ghana/2024-10/4-data-visualization.pdf)  

**Note:**  
- **Session 1** for Day 3 and Day 4 will be led by Debbie and Jane. These sessions will focus more on their experiences, reviewing the reports, and discussing what can be improved.  

Best,  
Mer

**Review by mariaruth — CHANGES_REQUESTED** [review]
@mariarrt94 overall, these look good. I am listing below specific comments for each session. the points with "-->" apply across sessions

### Day 1, Session 1
- --> suggest adding more opportunities for interaction within the presentation. asking for examples, asking if they think it would be similar in their context, etc. it's great that there are lots of examples, let's make it even more concrete by having them make linkages to ghana
- i understood the theme of the session to be 'what's your question' but i didn't understand how the exercise at the end relates to that exactly (no point at which they actually come up with a question they want to explore). perhaps i'll see this in later slides, but guidance now on how to come up with a good question for an analytical project could be useful (reasonable scope, measurable, etc)

### Day 1, session 2
- suggest using "government analytics" instead of data work in these slides to make the connection between the two sessions explicit (and because government analytics is the type of data work that will be most relevant for the audience
- --> suggest motivating each exercise with a question in plain english that explains what they could do using that function. for example for exercise 5, something like "say you want to know how many women work in the department" or "you want to do some gender analysis for the department, so you want to look at only the female staff members". it may prove helpful to benchmark this to excel functions, like this is the equivalent of using a filter in excel. 

### Day 2, Session 1
- following up on my comment for day 1 session 1, slide 10 here is what i was expecting to see there about formulating a good question
- i was confused by slide 20, why emphasize reasons for doing less government analytics? and the three explanations provided don't clearly seem like reasons for less analytics
- slide 28: this seems like a nice chance to have discussion about whether things are similar / different in Ghana
- slide 29, this feels out of flow to me but maybe i'm missing some connection

### day 2 session 2
- are slides 9-11 necessary given that this will be outside of WB? 
- slide 25 second bullet point has a couple of typos
- filtering: it seems kind of confusing to do the same task using different syntax on days 1 and 2. on day 1 you used subset to look at the women, now it's using filter. i think that will create confusion (or at least uncertainty as to when to do what). would be better to use only one function per type of task you introduce, to help people clearly map out what they should use when. the exercise here is quite clear so maybe do something different than subset on day 1?

### day 3 session 2
- slide 8. is the osf link correct? it goes to small business data for georgia, looks from the slides like you intend them to keep using the dept list data. 
- here (if not before) you will likely need to do an exercise for people to understand file paths. it's always an issue. in exercise 5 here you want people to export to documents, i think best to demonstrate how they find the filepath for the documents folder. but this may be relevant in an earlier day. 
- slide 46 "for further customization if you want" suggest referring back to the earlier discussion on reproducibility and noting that any manual changes will be difficult for others to reproduce. that then nicely motivates the next section which shows how to format using code. 

### day 4 session 2
- looks good!

**Comment by mariarrt94** [comment]
Thank you Maria, I am quoting your reply answering on the action I took to solve them. 
> @mariarrt94 overall, these look good. I am listing below specific comments for each session. the points with "-->" apply across sessions 
> 
> ### Day 1, Session 1
> * --> suggest adding more opportunities for interaction within the presentation. asking for examples, asking if they think it would be similar in their context, etc. it's great that there are lots of examples, let's make it even more concrete by having them make linkages to ghana

 ***Good idea. I am adding bubbles and specific spaces to link this to Ghana in all of the sessions***
> * i understood the theme of the session to be 'what's your question' but i didn't understand how the exercise at the end relates to that exactly (no point at which they actually come up with a question they want to explore). perhaps i'll see this in later slides, but guidance now on how to come up with a good question for an analytical project could be useful (reasonable scope, measurable, etc) 

***I added the slide from session 1 day 2 here, and linked it with the exercise***
> 
> ### Day 1, session 2
> * suggest using "government analytics" instead of data work in these slides to make the connection between the two sessions explicit (and because government analytics is the type of data work that will be most relevant for the audience

***Good idea, this is done****
> * --> suggest motivating each exercise with a question in plain english that explains what they could do using that function. for example for exercise 5, something like "say you want to know how many women work in the department" or "you want to do some gender analysis for the department, so you want to look at only the female staff members". it may prove helpful to benchmark this to excel functions, like this is the equivalent of using a filter in excel.
>

***Good idea, I am including the excel analogous in the exercises and also introducing them in plain english. ***
> ### Day 2, Session 1
> * following up on my comment for day 1 session 1, slide 10 here is what i was expecting to see there about formulating a good question 

***You are right, I moved this to the previous session, and here just a quick recap***
> * i was confused by slide 20, why emphasize reasons for doing less government analytics? and the three explanations provided don't clearly seem like reasons for less analytics 

***You are right, I removed this slide***

> * slide 28: this seems like a nice chance to have discussion about whether things are similar / different in Ghana 

***Added a bubble with the question to have a discussion at that point. Thank you for the suggestion***
> * slide 29, this feels out of flow to me but maybe i'm missing some connection 

***You are right, I forgot to delete this***
> 
> ### day 2 session 2
> * are slides 9-11 necessary given that this will be outside of WB?

- [ ] ***I am not sure, I was planning to have a meeting with Kpadam before the session. If they are not necessary I will remove them.

> * slide 25 second bullet point has a couple of typos

***This is Fixed***
> * filtering: it seems kind of confusing to do the same task using different syntax on days 1 and 2. on day 1 you used subset to look at the women, now it's using filter. i think that will create confusion (or at least uncertainty as to when to do what). would be better to use only one function per type of task you introduce, to help people clearly map out what they should use when. the exercise here is quite clear so maybe do something different than subset on day 1?
> 
***Changed the exercise in day 1 for unique***
> ### day 3 session 2
> * slide 8. is the osf link correct? it goes to small business data for georgia, looks from the slides like you intend them to keep using the dept list data.

- [ ] This is not correct pending to R correct link
> * here (if not before) you will likely need to do an exercise for people to understand file paths. it's always an issue. in exercise 5 here you want people to export to documents, i think best to demonstrate how they find the filepath for the documents folder. but this may be relevant in an earlier day.

***Good idea, I added it before teaching them how to read the data in session 2****
> * slide 46 "for further customization if you want" suggest referring back to the earlier discussion on reproducibility and noting that any manual changes will be difficult for others to reproduce. that then nicely motivates the next section which shows how to format using code.

***Added note on this***

> ### day 4 session 2
> * looks good!

Pending things

- [ ] Change the images of workflow to match our current datasets.
- [ ] Make sure they are csv (referred to as excel).
- [ ] Upload data to osf page.
- [ ] Confirm if slides about global paths are needed.
